src/policy/embed.rs

```diff
@@ -119,7 +119,7 @@ impl<'a, Pk: MiniscriptKey, Ctx: ScriptContext> TryFrom<&'a Miniscript<Pk, Ctx>>
                 Err(Error::ParseError("Public key hashes are not supported"))
             }
             Terminal::After(n) => Ok(Policy::After(n.0)),
-            Terminal::Older(n) => Ok(Policy::Older(n.0)),
+            Terminal::Older(n) => Ok(Policy::Older(n.0 as u16)),
```

Comment:
In d975984dd4b6db1fc27ae55c1a175c4d6964167b:

Why is this cast safe?

Reply:
I added a conversion similar to [to_relative_lock_time from rust-bitcoin](https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Sequence.html#method.to_relative_lock_time). Unfortunately this function does not exist in rust-elements.